### PR TITLE
[zh] Localize feature-gates/e*.md

### PR DESCRIPTION
**Pull request by 1000Delta**
Related to issue #44410 

Localize 19 feature-gates/e*.md:
* efficient-watch-resumption.md
* elastic-indexed-job.md
* enable-aggregated-discovery-timeout.md
* enable-equivalence-class-cache.md
* endpoint-slice-node-name.md
* endpoint-slice-proxying.md
* endpoint-slice-terminating-condition.md
* endpoint-slice.md
* ephemeral-containers.md
* even-pods-spread.md
* evented-pleg.md
* exec-probe-timeout.md
* expand-csi-volumes.md
* expand-in-use-persistent-volumes.md
* expand-persistent-volumes.md
* expanded-dns-config.md
* experimental-critical-pod-annotation.md
* experimental-host-user-namespace-defaulting.md
* external-policy-for-external-ip.md

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/elastic-indexed-job.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+---
+<!--
+Enables Indexed Jobs to be scaled up or down by mutating both
+`spec.completions` and `spec.parallelism` together such that `spec.completions == spec.parallelism`.
+See docs on [elastic Indexed Jobs](/docs/concepts/workloads/controllers/job#elastic-indexed-jobs)
+for more details.
+-->
+通过同时改变 `spec.completions` 和 `spec.parallelism`
+使得 `spec.completions == spec.parallelism` 来对带索引的 Job 执行扩容或缩容。
+有关详细信息，请参阅有关[弹性的带索引的 Job](/zh-cn/docs/concepts/workloads/controllers/job#elastic-indexed-jobs) 的文档。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/elastic-indexed-job.md:16]:
```suggestion
更多细节请参阅[弹性索引 Job](/zh-cn/docs/concepts/workloads/controllers/job#elastic-indexed-jobs) 文档。
```
technical docs 以简洁为美

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/endpoint-slice.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+_build:
+  list: never
+  render: false
+---
+<!--
+Enables EndpointSlices for more scalable and extensible
+ network endpoints. See [Enabling EndpointSlices](/docs/concepts/services-networking/endpoint-slices/).
+-->
+启用 EndpointSlice 以实现可扩缩性和可扩展性更好的网络端点。
+  参阅[启用 EndpointSlice](/zh-cn/docs/concepts/services-networking/endpoint-slices/)。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/endpoint-slice.md:15]:
```suggestion
参阅[启用 EndpointSlice](/zh-cn/docs/concepts/services-networking/endpoint-slices/)。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/ephemeral-containers.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  render: false
+---
+<!--
+Enable the ability to add
+{{< glossary_tooltip text="ephemeral containers" term_id="ephemeral-container" >}}
+to running Pods.
+-->
+启用添加
+{{< glossary_tooltip text="临时容器" term_id="ephemeral-container" >}}
+到正在运行的 Pod 的特性。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/ephemeral-containers.md:17]:
```suggestion
启用添加{{< glossary_tooltip text="临时容器" term_id="ephemeral-container" >}}到正在运行的
Pod 的特性。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/expand-csi-volumes.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+content_type: feature_gate
+
+_build:
+  list: never
+  render: false
+---
+<!--
+Enable the expanding of CSI volumes.
+-->
+启用扩展 CSI 卷。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/expand-csi-volumes.md:13]:
```suggestion
启用 CSI 卷的扩展。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/expand-in-use-persistent-volumes.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+_build:
+  list: never
+  render: false
+---
+<!--
+Enable expanding in-use PVCs. See
+[Resizing an in-use PersistentVolumeClaim](/docs/concepts/storage/persistent-volumes/#resizing-an-in-use-persistentvolumeclaim).
+-->
+启用扩充使用中的 PVC 的尺寸。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/expand-in-use-persistent-volumes.md:14]:
```suggestion
启用扩充使用中的 PVC 的大小。
```
跟下一句一致

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/expand-persistent-volumes.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+_build:
+  list: never
+  render: false
+---
+<!--
+Enable the expanding of persistent volumes. See
+[Expanding Persistent Volumes Claims](/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims).
+-->
+允许扩充持久卷。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/expand-persistent-volumes.md:14]:
```suggestion
允许扩展持久卷。
```
扩充、扩展 要统一吗

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/experimental-host-user-namespace-defaulting.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+<!--
+Enabling the defaulting user
+namespace to host. This is for containers that are using other host namespaces,
+host mounts, or containers that are privileged or using specific non-namespaced
+capabilities (e.g. `MKNODE`, `SYS_MODULE` etc.). This should only be enabled
+if user namespace remapping is enabled in the Docker daemon.
+-->
+启用主机默认的用户名字空间。
+这适用于使用其他主机名字空间、主机安装的容器，或具有特权或使用特定的非名字空间功能
+（例如 MKNODE、SYS_MODULE 等）的容器。
```
Comment (sftim) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/experimental-host-user-namespace-defaulting.md:17]: Optionally, update the localization; I'd expect:
```suggestion
（例如 `MKNODE`、`SYS_MODULE` 等）的容器。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/evented-pleg.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+(PLEG is an abbreviation for “Pod lifecycle event generator”).
+For this feature to be useful, you also need to enable support for container lifecycle events
+in each container runtime running in your cluster. If the container runtime does not announce
+support for container lifecycle events then the kubelet automatically switches to the legacy
+generic PLEG mechanism, even if you have this feature gate enabled.
+-->
+启用此特性后，kubelet 能够通过
+{{<glossary_tooltip term_id="cri" text="CRI">}}
+扩展从{{< glossary_tooltip text="容器运行时" term_id="container-runtime" >}}接收容器生命周期事件。
+（PLEG 是 `Pod lifecycle event generator` 的缩写，即 Pod 生命周期事件生成器）。
```
Comment (sftim) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/evented-pleg.md:21]: Optionally, avoid using backticks to surround the text (which is plain English, albeit full of jargon).

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/efficient-watch-resumption.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+_build:
+  list: never
+  render: false
+---
+<!--
+Allows for storage-originated bookmark (progress
+notify) events to be delivered to the users. This is only applied to watch operations.
+-->
+允许将存储发起的书签（进度通知）事件传递给用户。
+这仅适用于监视操作。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/efficient-watch-resumption.md:13]:
```suggestion
允许将存储发起的书签（进度通知）事件传递给用户。这仅适用于 watch（监视）操作。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/elastic-indexed-job.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+  list: never
+  render: false
+---
+<!--
+Enables Indexed Jobs to be scaled up or down by mutating both
+`spec.completions` and `spec.parallelism` together such that `spec.completions == spec.parallelism`.
+See docs on [elastic Indexed Jobs](/docs/concepts/workloads/controllers/job#elastic-indexed-jobs)
+for more details.
+-->
+通过同时改变 `spec.completions` 和 `spec.parallelism`
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/elastic-indexed-job.md:14]:
```suggestion
允许通过同时改变 `spec.completions` 和 `spec.parallelism`
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/endpoint-slice-node-name.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
...
+content_type: feature_gate
+
+_build:
+  list: never
+  render: false
+---
+<!---
+Enables EndpointSlice `nodeName` field.
+-->
+允许使用 EndpointSlice 的 `nodeName` 字段。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/endpoint-slice-node-name.md:13]:
```suggestion
启用 EndpointSlice 的 `nodeName` 字段。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/endpoint-slice-terminating-condition.md [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+---
+<!--
+Enables EndpointSlice `terminating` and `serving`
+ condition fields.
+-->
+允许使用 EndpointSlice 的 `terminating` 和 `serving` 条件字段。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/endpoint-slice-terminating-condition.md:12]:
```suggestion
启用 EndpointSlice 的 `terminating` 和 `serving` 状况字段。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/experimental-critical-pod-annotation.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+_build:
+  list: never
+  render: false
+---
+<!--
+Enable annotating specific pods as *critical*
+so that their [scheduling is guaranteed](/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/).
+This feature is deprecated by Pod Priority and Preemption as of v1.13.
+-->
+启用将特定 Pod 注解为 **critical** 的方式，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/experimental-critical-pod-annotation.md:15]:
```suggestion
启用将特定 Pod 注解为 **关键负载（Critical）** 的方式，
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/external-policy-for-external-ip.md [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+
+_build:
+  list: never
+  render: false
+---
+<!--
+Fix a bug where ExternalTrafficPolicy is not
+applied to Service ExternalIPs.
+-->
+修复一个 bug，该 bug 会导致 ExternalTrafficPolicy
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/external-policy-for-external-ip.md:14]:
```suggestion
修复一个缺陷，该缺陷会导致 ExternalTrafficPolicy
```

**Comment by windsonsea** [comment]
Need to rebase main. Some new changes are merged in #44478.

**Pull request merged**